website/docs/cli/configuration/configuration.mdx

```diff
@@ -24,15 +24,58 @@ Think of this file as where you [bootstrap the settings or configuration of your
 [terraform](/core-concepts/components/terraform), then [this is where](/cli/configuration/components#terraform-component-behavior)
 you'd specify the command to run (e.g. [`opentofu`](/core-concepts/projects/configuration/opentofu)),
 the base path location of the components, and so forth.
+## Base Path
 
-## Configuration File (`atmos.yaml`)
+The base path defines the root directory for components, stacks, workflows, and Atmos configurations. It can be set using the `ATMOS_BASE_PATH` environment variable or by passing the `--base-path` command-line argument. If both are set, the `--base-path` will take precedence. Both absolute and relative paths are supported. When a relative path is provided, it will be expanded to an absolute path relative to the current working directory, as Atmos executes commands that may change the working directory (e.g., Terraform).
+
+When the `base_path` is explicitly defined, it will always be resolved to the absolute path relative to the current working directory, if an absolute path was not already set.
+
+When the `base_path` is not explicitly provided or is empty, Atmos will attempt to infer the appropriate path using the following strategy, in order of precedence:
+1. If the current working directory contains any of the following files or directories: `atmos.yaml`, `.atmos.yaml`, `.github/atmos.yaml`, or `atmos.d/**/*`, Atmos will use the directory containing these files as the `base_path`.
```

Comment:
To be clear, it's the prefix of the filename, not the dirname.

Reply:
I think `.github/atmos.yaml` and `.atmos.yaml` from the *root* of the repo, but not any subpath.